docker-compose.yml

```diff
@@ -1106,6 +1106,20 @@ services:
       - EX_APPS_NET=${EX_APPS_NET:-ipv4@localhost}
       - EX_APPS_COUNT=${EX_APPS_COUNT:-50}
 
+  appapi-dsp-harp:
+    # todo
+    image: nextcloud-appapi-harp:latest
```

Comment:
I assume that would be added here as well? Or is there a plan to have an image upstream in harp

Reply:
not sure what you refer to but yeah the image would be publicly available, will change to ghcr.io image